app/http.php

```diff
@@ -285,26 +307,25 @@ function getStorageDevice(string $root): Device
     }
 }
 
-function removeAllRuntimes(Table $activeRuntimes, Orchestration $orchestration): void
+function cleanUp(Table $activeRuntimes, Orchestration $orchestration, array $networks = []): void
 {
-    Console::log('Cleaning up containers...');
+    Console::log('Cleaning up containers and networks...');
 
     $functionsToRemove = $orchestration->list(['label' => 'openruntimes-executor=' . System::getHostname()]);
 
     if (\count($functionsToRemove) === 0) {
         Console::info('No containers found to clean up.');
     }
 
-    $callables = [];
-
+    $runtimesToRemove = [];
     foreach ($functionsToRemove as $container) {
-        $callables[] = function () use ($container, $activeRuntimes, $orchestration) {
+        $runtimesToRemove[] = function () use ($container, $activeRuntimes, $orchestration) {
             try {
                 $orchestration->remove($container->getId(), true);
 
                 $activeRuntimeId = $container->getName();
 
-                if (!$activeRuntimes->exists($activeRuntimeId)) {
+                if ($activeRuntimes->exists($activeRuntimeId)) {
```

Comment:
Previously it was `!$activeRuntimes->exists`. Was that a bug?

Reply:
I think its a bug.. because if the activeRuntimeId does not exist in the swoole table, how can we delete it? 

But Ive reverted it since I want to validate with matej